cli.md

```diff
@@ -34,9 +34,12 @@ File group(s) used as output.
 
 ## Optional parameters
 
-### `-g, --group-id GROUPID`
+### `-g, --group-id ID`
 
-The `mets:file@GROUPID` to process. If no groupID is defined all files within the `inputGrp` will be processed. Repeatable and multiple IDs can be provided separated by comma.
+The `ID` of the `mets:div[@TYPE='page']` that contains the pointers to files
```

Comment:
Alternatively: The `mets:div[@TYPE='page']@ID` that

Reply:
If so, then `mets:div[@TYPE='page']/@ID` so it's a proper XPath. No preference as long as it's consistent.